sktime/forecasting/base/tests/test_fh.py

```diff
@@ -323,10 +334,11 @@ def test_absolute_to_absolute_with_integer_horizon(freqstr):
     """Test converting between absolute and relative with integer horizon."""
     # Converts from absolute to relative and back to absolute
     train = pd.Series(1, index=pd.date_range("2021-10-06", freq=freqstr, periods=3))
+    cutoff = get_cutoff(train, return_index=True)
```

Comment:
That's what I was afraid of. You are forcing a user to use `get_cutoff` function. You could also force a user to use `pd.period_range` above. Or force him/her to pass `freq`? Why is this choice better?

Reply:
Two arguments:
1. this is not actually a user will have to do, typically.

If they *specify* the forecasting horizon, they will either do one of:
* direct construction, absolute timestamps -> has freq
* direct construction, relatie timestamps -> has freq
* direct construction, integers -> does not have freq, but will be interpreted by forecaster

and not call one of the conversion methods.

2. passing `freq` instead is also possible, as per your interface.

Reply:
we could also test for both, i.e., passing `cutoff` and `freq`?

Reply:
I hope you do not mean `pd.Timestamp` when you say "timestamps" ;)

Reply:
> I hope you do not mean `pd.Timestamp` when you say "timestamps" ;)

I fear that would be a user expectation...

Reply:
> Sure, the more tests the merrier :D

I added a test where `freq` is passed to `ForecastingHorizon`

Reply:
> I fear that would be a user expectation...

Exactly. And in this case we should push a user to other means of passing frequency information.